app/controllers/comments_controller.rb

```diff
@@ -202,6 +203,61 @@ def preview
     end
   end
 
+  ## TODO: Service-orient this method - SubscriptionCreator.call(...)
+  # rubocop:disable Metrics/PerceivedComplexity, Metrics/CyclomaticComplexity
+  def subscribe
+    skip_authorization
+
+    comment_id = permitted_attributes(Comment)[:comment_id]
+    article_id = permitted_attributes(Comment)[:article_id]
+    comment = Comment.find(comment_id) if comment_id.present?
+    article = Article.find(article_id) if article_id.present?
+    notification_id = permitted_attributes(Comment)[:notification_id]
+    notification = NotificationSubscription&.find(notification_id) unless notification_id.nil?
```

Comment:
Minor, but can we change this param from `notification_id` to `subscription_id` — it feels like it's already hard enough to distinguish between `Notification` and `NotificationSubscription`, Naming Things :tm: might help.

Also: `NotificationSubscription&.find` is an unusual syntax — is there a scenario where `NotificationSubscription` is `nil`? The `find` should probably be scoped to `current_user` (the scenario is extremely unlikely, but we would not want someone to "spoof" another user and destroy their subscriptions). Maybe that's where the `&.find` came from originally?